examples/src/main/resources/categories.txt

```diff
@@ -0,0 +1,1000 @@
+0: tench, Tinca tinca
```

Comment:
is it the same as imagenet? if it is ,we should re-use the one we have in model zoo

Reply:
Yes, they are the same labels as ImageNet.